Hash mismatch on axios when installing with a lockfile

* **Node Version**: v8.9.1
* **Npm Version**: 5.5.1
* **Yarn Version**: 1.3.2
* **Platform**: CircleCI v1 (Ubuntu 14.04)
* **Package Version**: 5.0.2 and 5.0.3

Since https://github.com/contentful/contentful.js/pull/195 we get the following lockfile hash mismatch when installing on CircleCI:

```
$yarn install --frozen-lockfile
yarn install v1.3.2
[1/5] Validating package.json...
[2/5] Resolving packages...
⠁ [3/5] Fetching packages...
error https://github.com/contentful/axios/archive/fix/https-via-http-proxy.tar.gz: Fetch succeeded for undefined. However, extracting "https://github.com/contentful/axios/archive/fix/https-via-http-proxy.tar.gz" resulted in hash "76fe9fcbb0e5907608bf2fa55d01846493474f48", which did not match the requested hash "bb534c3cb2894129e223c56f1bdad574a7c9f1b7".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

yarn install --frozen-lockfile returned exit code 1

Action failed: yarn install --frozen-lockfile
```

When using forked libraries like this, these and related issues can be avoided by publishing the forked version to the npm registry under a scope (e.g., `@contentful/axios`) or other name (e.g., `axios-contentful-fix`).  If you would like, I am happy to create the package and open a PR to use it here, just LMK. Thanks!

Thanks! Looks like I am getting the following while building a bundle:

```
ERROR in ./node_modules/contentful/dist/es-modules/contentful.js
       Module not found: Error: Can't resolve 'axios' in '/tmp/build_432a5fc110f712411fba1437061d6d31/homezen-hz-webapp-506bb01c87a66341cf7e596cd60b022220d05003/node_modules/contentful/dist/es-modules'
       @ ./node_modules/contentful/dist/es-modules/contentful.js 10:0-26
       @ ./app/contentful/storage-client.js
       @ ./app/contentful/contentful.js
       @ ./app/contentful/index.js
       @ ./app/client-main.js
       @ multi babel-polyfill ./app/client-main
```

--- a/lib/contentful.js
+++ b/lib/contentful.js
@@ -5,7 +5,7 @@
  * @see ContentfulClientAPI
  */
 
-import axios from 'axios'
+import axios from '@contentful/axios'
 import {createHttpClient, getUserAgentHeader} from 'contentful-sdk-core'
 import createContentfulApi from './create-contentful-api'
 import createLinkResolver from './create-link-resolver'

--- a/package.json
+++ b/package.json
@@ -121,7 +121,7 @@
   "bundlesize": [
     {
       "path": "./dist/contentful.browser.js",
-      "maxSize": "38Kb"
+      "maxSize": "39Kb"
     },
     {
       "path": "./dist/contentful.browser.min.js",
@@ -133,7 +133,7 @@
     },
     {
       "path": "./dist/contentful.legacy.min.js",
-      "maxSize": "22Kb"
+      "maxSize": "23Kb"
     },
     {
       "path": "./dist/contentful.node.js",

--- a/test/unit/contentful-test.js
+++ b/test/unit/contentful-test.js
@@ -17,7 +17,7 @@ test('Throws if no space is defined', (t) => {
   t.end()
 })
 test('Generate the correct User Agent Header', (t) => {
-  createClientRewireApi.__Rewire__('axios', sinon.stub)
+  createClientRewireApi.__Rewire__('@contentful/axios', sinon.stub)
   const createHttpClientStub = sinon.stub()
   const rateLimitStub = sinon.stub()
   createClientRewireApi.__Rewire__('createHttpClient', createHttpClientStub)
@@ -34,11 +34,11 @@ test('Generate the correct User Agent Header', (t) => {
 
   createClientRewireApi.__ResetDependency__('rateLimit')
   createClientRewireApi.__ResetDependency__('createHttpClient')
-  createClientRewireApi.__ResetDependency__('axios')
+  createClientRewireApi.__ResetDependency__('@contentful/axios')
   t.end()
 })
 test('Passes along HTTP client parameters', (t) => {
-  createClientRewireApi.__Rewire__('axios', sinon.stub)
+  createClientRewireApi.__Rewire__('@contentful/axios', sinon.stub)
   createClientRewireApi.__Rewire__('version', 'version')
   const createHttpClientStub = sinon.stub()
   const rateLimitStub = sinon.stub()
@@ -49,7 +49,7 @@ test('Passes along HTTP client parameters', (t) => {
   t.ok(createHttpClientStub.args[0][1].headers['X-Contentful-User-Agent'])
   createClientRewireApi.__ResetDependency__('rateLimit')
   createClientRewireApi.__ResetDependency__('createHttpClient')
-  createClientRewireApi.__ResetDependency__('axios')
+  createClientRewireApi.__ResetDependency__('@contentful/axios')
   t.end()
 })
 

